app/views/RoomView/index.js

```diff
@@ -131,6 +141,14 @@ export default class RoomView extends LoggedView {
 				}
 			}
 			safeAddListener(this.rooms, this.updateRoom);
+			safeAddListener(this.chats, () => {
+				const unreadsCount = this.chats.filtered('(unread > 0 || alert == true)').length;
+				if (unreadsCount !== navigation.getParam('unreadsCount')) {
+					navigation.setParams({
+						unreadsCount
```

Comment:
Unread count should consider the sum of `unread` and `userMentions` of all subscriptions.

Reply:
You can compare against web badge.
Example:
<img src="https://user-images.githubusercontent.com/804994/56319226-7bd28300-6137-11e9-9f8b-f443685167ac.png" with="350" />